tools/afl-fuzzer/platform/byt-host.c

```diff
@@ -0,0 +1,564 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
i wonder if maybe we should restructure this so that both fuzzers can access a common platform library to avoid duplicating code.